hack/install-kcp-with-plugins.sh

```diff
@@ -113,16 +117,15 @@ fi
 
 if [ -d "$kcp_folder" ]; then :
 elif [ "$kcp_create_folder" == "true" ]; then
-    if [ $verbose == "true" ] ; then
+    if [ $verbose == "true" ]; then
         echo "Creating folder: $kcp_folder"
     fi
     mkdir -p "$kcp_folder"
 else
     echo "Specified folder does not exist: $kcp_folder" >&2; exit 1;
 fi
 
-if [ $verbose == "true" ]
-then
+if [ $verbose == "true" ]; then
     echo "Downloading KCP $kcp_version $kcp_os/$kcp_arch..."
     curl -SL -o kcp.tar.gz "https://github.com/kcp-dev/kcp/releases/download/${kcp_version}/kcp_${kcp_version//v}_${kcp_os}_${kcp_arch}.tar.gz"
```

Comment:
Is that `//v` just an oblique way of prefixing "v"?  If so then I recommend doing this more directly: put the "v" in the literal part of the string.

Reply:
OK, I found documentation of the parameter expansion trick used here.  It is not precisely targeted:

```console
bash-5.1$ export version=v0.2.0-preview

bash-5.1$ echo ${version//v}
0.2.0-preiew
```

I think you want the following instead.

```console
bash-5.1$ echo ${version/#v/}
0.2.0-preview
```